mne/io/reference.py

```diff
@@ -197,6 +197,9 @@ def add_reference_channels(inst, ref_channels, copy=True):
     else:  # n_ref_channels == 1 and a single ref digitization exists
         ref_dig_array = np.concatenate((ref_dig_loc[0]['r'],
                                        ref_dig_loc[0]['r'], np.zeros(6)))
+        # Replace the (possibly new) Ref location for each channel
```

Comment:
@Eric89GXL I believe you! I guess I didn't grep the right string. Don't bother digging anything up for me, I'll try to figure it out myself. Though I cannot imagine what fwd code would  need to care about the ref, since everything is CAR there...